tests/helpers/MemoryTransport.ts

```
@@ -71,11 +76,21 @@
   onClose(handler: () => void): void {
     this.onCloseHandler = handler;
   }
   
   onError(handler: (error: Error) => void): void {
     this.onErrorHandler = handler;
   }
+  
+  // Required by Transport interface
+  async start(): Promise<void> {
+    // No-op for memory transport
+  }
```

Biome (1.9.4)

[error] Duplicate class member name "start"



(lint/suspicious/noDuplicateClassMembers)